gnome-shell/src/gnome-shell-sass/widgets/_login-lock.scss

```diff
@@ -224,32 +223,32 @@ $_gdm_dialog_width: 25em;
 }
 
 .web-login-spinner {
-  background-color: rgba(0, 0, 0, 0.5);
-  border: 5px rgba(0, 0, 0, 0.0);
+  color: $osd_fg_color;
```

Comment:
Looks good except this I'm not sure. Did this change to change it's value for light/dark ?
Back in the days it was always white

Reply:
Yes, it's always white and it has to be white here because in both cases it's an osd-like spinner:

![image](https://github.com/ubuntu/yaru/assets/345675/11aac5cf-6c4c-47ac-b807-c4b67b307d5c)

And when the dialog is white it has still to be white on gray.